Libraries/zephyr/MAX/Include/wrap_max32_tmr.h

```diff
@@ -142,13 +142,16 @@ static inline int Wrap_MXC_TMR_GetClockIndex(int z_clock)
         return MXC_TMR_EXT_CLK;
     case 2: // ADI_MAX32_PRPH_CLK_SRC_IBRO
         return MXC_TMR_8M_CLK;
+#if !defined(CONFIG_SOC_MAX78002)
     case 3: //ADI_MAX32_PRPH_CLK_SRC_ERFO
         return MXC_TMR_32M_CLK;
+#endif
     case 4: //ADI_MAX32_PRPH_CLK_SRC_ERTCO
         return MXC_TMR_32K_CLK;
     case 5: //ADI_MAX32_PRPH_CLK_SRC_INRO
         return MXC_TMR_INRO_CLK;
-#if defined(CONFIG_SOC_MAX32690)
+#if defined(CONFIG_SOC_MAX32655) || (CONFIG_SOC_MAX32680) || (CONFIG_SOC_MAX32690) || \
+    (CONFIG_SOC_MAX78002)
     case 6: //ADI_MAX32_PRPH_CLK_SRC_ISO
         return MXC_TMR_ISO_CLK;
 #endif
```

Comment:
This PR content shall be just focus on MAX780002, send fix or improvement related things as separate commit & PR.

Reply:
Updated. Created https://github.com/analogdevicesinc/msdk/pull/1169 PR for others.